docs/concepts/what-is-a-query.md

```diff
@@ -21,38 +27,12 @@ Queries are available to expose the internal Workflow Execution state to the ext
   Query handling logic must be **read-only** and cannot change the Workflow Execution state in any way, or contain any blocking code.
   This means that Query handling logic can not spawn Activity Executions.
 
+Queries are strongly consistent and will operate with the last available state.
```

Comment:
I would try to save a little more of the original information here. Maybe something like
"
Queries are strongly consistent. A strongly consistent Query is guaranteed to return the state, which includes all Events that came in and were confirmed by the returned response before the Query was issued.
"

Reply:
Considering the feedback - I did a quick rewrite of the whole thing.

LMK what you think -